tests/templates/kuttl/oidc/login.py

```diff
@@ -24,7 +24,11 @@
     else f"http://{keycloak_service}:8080"
 )
 
-if nifi_version in ["2.0.0-M4"]:
+# startswith instead of an exact check to
+# a) hit all 2.x versions and
+# b) to allow for custom images because `nifi_version` will contain the whole custom image string
+# e.g. 2.0.0-M4,localhost:5000/stackable/nifi:2.0.0-M4-stackable0.0.0-dev
+if nifi_version.startswith("2"):
```

Comment:
What about `not nifi_version.startswith("1.")` to not break on e.g. NiFi 3?
We could also swap the order of the if branches

Reply:
I'm fine with those changes. @labrenbe offered to take over this PR so I'd leave it to him if that's okay.